HierarchicalEOM.jl/cavityQED.qmd

```diff
@@ -0,0 +1,255 @@
+---
+title: "Cavity QED system"
+author: Shen-Liang Yang, Yi-Te Huang
+date: last-modified
+date-format: iso
+
+engine: julia
+---
+
+## Package Imports
+```{julia}
+using HierarchicalEOM
+using LaTeXStrings
+import Plots
```

Comment:
I would use CairoMakie instead. There is also WGLMakie to make it JavaScript interactive, but I don't know if this works in quarto already.